docker/templates/Dockerfile.erb

```diff
@@ -1,148 +1,147 @@
 # This Dockerfile was generated from templates/Dockerfile.erb
-<% if local_artifacts == 'false' -%>
-<%   url_root = 'https://artifacts.elastic.co/downloads/logstash' -%>
-<% else -%>
-<%   url_root = 'http://localhost:8000' -%>
-<% end -%>
-<% if image_flavor == 'oss' -%>
-<%   tarball = "logstash-oss-#{elastic_version}-linux-$(arch).tar.gz" -%>
-<%   license = 'Apache 2.0' -%>
-<% else -%>
-<%   tarball = "logstash-#{elastic_version}-linux-$(arch).tar.gz" -%>
-<%   license = 'Elastic License' -%>
-<% end -%>
-<% if image_flavor == 'ubi8' %>
-<%   base_image = 'docker.elastic.co/ubi8/ubi-minimal' -%>
-<%   package_manager = 'microdnf' -%>
-# Minimal distributions do not ship with en language packs.
-<%   locale = 'C.UTF-8' -%>
-<% elsif image_flavor == 'ironbank' -%>
-<%   package_manager = 'yum' -%>
-<% else -%>
-<%   base_image = 'ubuntu:20.04' -%>
-<%   package_manager = 'apt-get' -%>
-<%   locale = 'en_US.UTF-8' -%>
-<% end -%>
 <% if image_flavor == 'ironbank' -%>
-ARG BASE_REGISTRY=registry1.dsop.io
+ARG BASE_REGISTRY=registry1.dso.mil
```

Comment:
Can we indent the `ironbank` image definition?

Reply:
Do you mean add \t to all lines inside `if image_flavor ==  'ironbank'` block?
The output Dockerfile will result in indentation as well. I think we don't want it, isn't it?